examples/aws/resourcerecordset/v1alpha1/claim-using-zoneidref.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: route53.aws.crossplane.io/v1alpha1
+kind: ResourceRecordSet
```

Comment:
Shouldn't we also be using AWSResourceRecordSetV1alpha1 here too?

Reply:
you're right!